CHANGELOG.md

```diff
@@ -6,19 +6,27 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 None
+<a name="3.0.5"</a>
+## [3.0.5] 2021-12-6
+### Added
```

Comment:
the convention for this kind of update would be `### Changed` https://keepachangelog.com/en/1.0.0/